lib/board.rb

```diff
@@ -0,0 +1,47 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    self.cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    #puts " #{cells[0]} | #{cells[1]} | #{cells[2]} \n-----------\n #{cells[3]} | #{cells[4]} | #{cells[5]} \n-----------\n #{cells[6]} | #{cells[7]} | #{cells[8]}  \n"
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(num)
+    cells[num.to_i - 1]
+  end
+
+  def full?
+    turn_count == 9
+  end
+
+  def turn_count
+    cells.count("X") + cells.count("O")
+  end
+
+  def valid_move?(num)
+     taken?(num) == false && num.to_i.between?(1, 9)
```

Comment:
Instead of comparing to `false`, you can negate the result of the boolean operation with the `!` (bang) operator.
```
!taken?(num)
```